test/models/multiple-results-event.tests.js

```diff
@@ -0,0 +1,427 @@
+      input.numOfResults = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "numOfResults must be a Number");
+    });
+
+    it('It should throw if numOfResults is object', () => {
+      input.numOfResults = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "numOfResults must be a Number");
+    });
+
+    it('It should throw if betEndTime is missing', () => {
+      delete input.betEndTime;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if betEndTime is null', () => {
+      input.betEndTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if betEndTime is undefined', () => {
+      input.betEndTime = undefined;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if betEndTime is number', () => {
+      input.betEndTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if betEndTime is array', () => {
+      input.betEndTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if betEndTime is object', () => {
+      input.betEndTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is missing', () => {
+      delete input.resultSetStartTime;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is null', () => {
+      input.resultSetStartTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is undefined', () => {
+      input.resultSetStartTime = undefined;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is number', () => {
+      input.resultSetStartTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is array', () => {
+      input.resultSetStartTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is object', () => {
+      input.resultSetStartTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is null', () => {
```

Comment:
need to add another test `it does not validate betStartTime if undefined`.

Reply:
these two are in format test, if not passing these two field, format should still pass.